lib/compiler.js

```diff
@@ -233,7 +233,7 @@
     },
 
     format: function(context, node) {
-      return '.w(' + escape(node[1] + node[2]) + ')';
+      return '.w(' + escape(node[1]) + ')';
```

Comment:
I really hate that our parser gives us things like `node[0]` and `node[1]` instead of `node.body` and `node.lineNo`. Tornado...